mapping/jsonld-contexts/mapping.jsonld

```diff
@@ -19,6 +19,7 @@
 		"qoS": "https://ontology.eglobalmark.com/qoS",
 		"subAttribute1": "https://ontology.eglobalmark.com/subAttribute1",
 		"subAttribute2": "https://ontology.eglobalmark.com/subAttribute2",
-		"subAttribute3": "https://ontology.eglobalmark.com/subAttribute3"
+		"subAttribute3": "https://ontology.eglobalmark.com/subAttribute3",
+		"specificAccessPolicy": "https://ontology.eglobalmark.com/egm#specificAccessPolicy"
```

Comment:
@bobeal the egm context conains many other concepts  (such as sensor, device....) that may cause some conflits. So, I added the specificAcceessPolicy attribute manually to the mapping context. Is that Ok ?

Reply:
ah.

so maybe one cleaner way to solve this would be to move the `specificAccessPolicy` from the EGM cross-context to the authorization specific context (https://github.com/easy-global-market/ngsild-api-data-models/blob/master/authorization/jsonld-contexts/authorization.jsonld).